docs/guides/qiskit-transpiler-service.mdx

```diff
@@ -96,9 +96,9 @@ The following examples demonstrate how to transpile circuits using the Qiskit Tr
 
 ## Limits of the Qiskit Transpiler Service
 
-Following are the most relevant limitations of the service:
+Following is the most relevant limitations of the service:
```

Comment:
Change 'is' to 'are' in 'Following is the most relevant limitations of the service' for grammatical correctness